src/index.pug

```diff
@@ -7,6 +7,10 @@ html
       body {
         background-color: #eee !important;
       }
+    script.
```

Comment:
Maybe are you talking about `DefinePlugin`? Because `ProvidePlugin` will load the module but not create a global variable.